reference/docs-conceptual/community/contributing/pull-requests.md

```diff
@@ -0,0 +1,90 @@
+---
+title: How to submit pull requests
+description: This article explains how to submit pull requests to the PowerShell-Docs repository.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# How to submit pull requests
+
+To make changes to content, submit a pull request (PR) from your fork. A pull request must be reviewed
+before it can merged. For best results, review the [editorial checklist](editorial-checklist.md)
+before submitting your pull request.
+
+## Target the correct branch
+
+All pull requests should target the `staging` branch. Changes should never be submitted to the
+`live` branch. Changes made in the `staging` branch get merged into `live`, overwriting any changes
+made to `live`.
+
+If you are submitting a change that only applies to an unreleased version of PowerShell, check for a
+release branch for that version. Your should be targeted at the release branch. Release branches
+have the following name pattern: `release-<version>`.
+
+## Make the pull request process work better for everyone
+
+The simpler and more focused you can make your PR, the faster it can be reviewed and merged.
+
+### Avoid branches that update large numbers of files or contain unrelated changes
+
+Avoid creating PRs that contain unrelated changes. Separate minor updates to existing articles from
+new articles or major rewrites. Work on these changes in separate working branches.
+
+Bulk changes create PRs with large numbers of changed files. Limit your PRs to a maximum of 50
+changed files. Large PRs are difficult to review and are more prone to contain errors.
+
+### Renaming or deleting files
+
+If you are renaming or deleting files, there must be an issue associated with the PR. That issue
+must discuss the need to rename or delete the files.
+
+Avoid mixing content additions or change with file renames and deletes. Any file that is renamed or
+deleted must be added to the master redirection file. When possible, you should also update any
+files that link to the renamed or deleted content. This includes any TOC files.
+
+## Docs PR validation service
+
+The Docs PR validation service is a GitHub app that runs validation rules on the files in a PR. You
+must fix any errors or warnings (see exceptions) reported by the validation service.
+
+The following warnings can be ignored:
+
+- > Can't find service name for `<version>/<modulepath>/About/About.md`
+
+-  > Metadata with following name(s) are not allowed to be set in Yaml header, or as file level
+   > metadata in docfx.json, or as global metadata in docfx.json: `locale`. They are generated by
+   > Docs platform, so the values set in these 3 places will be ignored. Please remove them from all
+   > 3 places to resolve the warning.
```

Comment:
Same comment, these should likely be codefenced instead of listitem to bubble.

Reply:
Changed